src/utils/instrumentation.ts

```diff
@@ -154,6 +154,22 @@ function validateAndUpdateInstrumentation(
  * @returns {string} The truncated value.
  */
 function truncateValue(value: string, maxLen: number) {
```

Comment:
Mind it has been a while since I've touched a TypeScript codebase but I'm curious about this `value` parameter here. Should it maybe be tweaked to indicate that it also accepts an object with a value property? Something similar to: https://github.com/googleapis/nodejs-logging/blob/42859ab634721cb57ca76960f8a8aa5f6dd3a987/src/index.ts#L440

Reply:
Makes sense, changed - thanks @ruyadorno!